package.json

```diff
@@ -16,6 +16,13 @@
 	"description": "An ES7/ES2016 spec-compliant `Array.prototype.includes` shim/polyfill/replacement that works as far down as ES3.",
 	"license": "MIT",
 	"main": "index.js",
+	"files": [
```

Comment:
The “files” field is dangerous - the only way published files should ever be restricted is by npmignore.